client/src/components/startpage/Home.jsx

```diff
@@ -1,37 +1,46 @@
-import React from 'react'
+import React from "react";
 
 export default class Home extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {name: ''};
+    this.state = { name: "" };
+    this.state = { validInput: true };
 
     this.handleChange = this.handleChange.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
-    }
+  }
 
-    handleChange(event) {
-      this.setState({name: event.target.value});
-    }
+  handleChange(event) {
+    this.setState({ name: event.target.value });
+  }
 
-    handleSubmit() {
-      const name = this.state.name;
-      const letters = /^[A-Öa-ö]+$/;
-      if (name.length > 0 && name.match(letters)) {
-        this.props.history.push('/lobby', { name});
-      } else {
-        alert('You did not enter a name! Only letters allowed.');
-      }
+  handleSubmit() {
+    const name = this.state.name;
+    const letters = /^[A-Öa-ö]+$/;
+    if (name.length > 0 && name.match(letters)) {
+      this.props.history.push("/lobby", { name });
+    } else {
+      this.setState({ validInput: false });
     }
+  }
 
-    render() {
-      return(
+  render() {
+    if (this.state.validInput === true) {
+      return (
         <form onSubmit={this.handleSubmit}>
           <label>
             Enter name:
-            <input type="text" value={this.state.name} onChange={this.handleChange} />
+            <input
+              type="text"
+              value={this.state.name}
+              onChange={this.handleChange}
+            />
           </label>
           <input type="submit" value="Submit" />
         </form>
       );
+    } else {
+      return <h1>FEL INPUT! (Hur gör jag detta snyggare..)</h1>;
```

Comment:
Såhär skulle du kunna skriva render funktionen för att få det snyggare. (jag har inte testat att köra detta, så kanske inte funkar dirr)
`
render(){
   return(
      <div>
         <form>
         ...
         </form>
         { !this.state.validInput &&
            <h1>FEL INPUT! </h1>
         }
      </div>
   )
}`